### DISCUSSION TITLE
Using wrapper with non-AWS databases

### DISCUSSION THREAD

**Discussion by mnoky** [aws/aws-advanced-jdbc-wrapper#710] [General]
We're in the process of migrating a bunch of Spring/Spring-Boot apps to run in AWS EKS and use an RDS database. We'll be using the Advanced AWS JDBC wrapper to handle with IAM-based authentication.

My question: Are there any problems (performance or otherwise) with using the Wrapper in our current environment (non-RDS Postgres database)? That way, we can build our application code and use it now, then just update the config when moving to EKS/RDS. Of course we wouldn't configure the IAM plugin for now, we'd just update the JDBC URL to include "aws-wrapper". This seems to work fine, but I just want to make sure this is an acceptable practice.

**Comment by crystall-bitquill** [comment]
Hi, @mnoky, thanks for your question. Yes, that is an acceptable practice, the AWS JDBC Driver should be able to be used with non-RDS PostgreSQL databases.
